examples/user_guide/Network_Graphs.ipynb

```diff
@@ -0,0 +1,335 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interactive features"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Hover and selection policies\n",
+    "\n",
+    "Thanks to Bokeh we can reveal more about the graph by hovering over the nodes and edges. The ``Graph`` element provides an ``inspection_policy`` and a ``selection_policy``, which define whether hovering and selection highlight edges associated with the selected node or nodes associated with the selected edge, these policies can be toggled by setting the policy to ``'nodes'`` and ``'edges'``."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [tools=['hover']]\n",
+    "bezier_graph.opts(plot=dict(inspection_policy='edges'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to changing the policy we can also change the colors used when hovering and selecting nodes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [tools=['hover', 'box_select']] (edge_hover_line_color='green' node_hover_fill_color='red')\n",
+    "bezier_graph.opts(plot=dict(inspection_policy='nodes'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Additional information\n",
+    "\n",
+    "We can also associate additional information with the nodes of a graph. By constructing the ``Nodes`` explicitly we can declare an additional value dimension, which we can reveal when hovering and color the nodes by:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [tools=['hover'] color_index='Type'] (cmap='Set1')\n",
+    "nodes = hv.Nodes((x, y, node_indices, ['Output']+['Input']*(N-1)),  vdims=['Type'])\n",
+    "hv.Graph(((source, target), nodes, paths)).redim.range(**padding)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Working with NetworkX"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "NetworkX is a very useful library when working with network graphs and the Graph Element provides ways of importing a NetworkX Graph directly. Here we will load the Karate Club graph and use the ``circular_layout`` function provided by NetworkX to lay it out:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [tools=['hover']]\n",
+    "import networkx as nx\n",
+    "G = nx.karate_club_graph()\n",
+    "hv.Graph.from_networkx(G, nx.layout.circular_layout).redim.range(**padding)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Animating graphs"
+   ]
```

Comment:
Animating graphs sounds like a great thing to include, but I don't see anything in this section about that.  Can you add something that actually does animate a graph, e.g. while dragging a slider?

Reply:
Sure, I can add a different example.

Reply:
This one also could use the padding that helps the ones above.  I guess auto-padding is only for 2.0?

Reply:
Unfortunately that's probably the case.